tools/devtool

```diff
@@ -972,6 +972,16 @@ cmd_build() {
     return $ret
 }
 
+function cmd_make_release {
```

Comment:
As far as I understand it, one would firstly call prepare_release from outside devtool and then have the choice to do the actual release by calling the ./tools/release.sh from either devtool or from outside?

Reply:
The process would be to just call `prepare-release.sh`, do the PR, tag, push etc and then the release automation will take it from there. I.e. there's no need to call `release.sh` directly.